eng/Xunit3/Xunit3.targets

```diff
@@ -1,9 +1,6 @@
 <Project>
   <ItemGroup>
     <PackageVersion Include="xunit.v3" Version="$(XunitV3Version)" />
-    <PackageVersion Include="xunit.v3.core" Version="$(XunitV3Version)" />
```

Comment:
Could you please help me understand why the changes here are needed? And the `PackageReference` ones in the renamed project file.

Reply:
Thank you for the explanation!